### ISSUE TITLE
DockerTests.test010Install failing due to incorrect file permission assertion

### ISSUE THREAD

**Issue by mark-vieira** [elastic/elasticsearch#60853]
**Build scan**:
https://gradle-enterprise.elastic.co/s/5iue6ecykwawg/tests/:qa:os:destructiveDistroTest.default-docker/org.elasticsearch.packaging.test.DockerTests/test010Install#1

**Repro line**:
Seems packaging tests don't produce repro lines.

**Reproduces locally?**:
Yes, it did [reproduce locally for me](https://gradle-enterprise.elastic.co/s/5vszfsstjqtcy).

**Applicable branches**:
Looks to be specific to `7.x`.

**Failure history**:
https://gradle-enterprise.elastic.co/scans/tests?search.buildToolTypes=gradle&search.buildToolTypes=maven&search.relativeStartTime=P7D&search.timeZoneId=America/Los_Angeles&tests.container=org.elasticsearch.packaging.test.DockerTests&tests.disabledDistributions=WyJvdXRjb21lOnNraXBwZWQiXQ&tests.sortField=FAILED&tests.test=test010Install&tests.unstableOnly=true

**Failure excerpt**:
```
org.elasticsearch.packaging.test.DockerTests > test010Install FAILED
    java.lang.AssertionError: File group of /usr/share/elasticsearch/config/elasticsearch.keystore is wrong
    Expected: "root"
         but: was "elasticsearch"
        at __randomizedtesting.SeedInfo.seed([DEB273B486698D2:29B487A82ACBC78A]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.elasticsearch.packaging.util.Docker.assertPermissionsAndOwnership(Docker.java:393)
        at org.elasticsearch.packaging.util.Docker.lambda$verifyOssInstallation$5(Docker.java:436)
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
        at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
        at org.elasticsearch.packaging.util.Docker.verifyOssInstallation(Docker.java:436)
        at org.elasticsearch.packaging.util.Docker.verifyContainerInstallation(Docker.java:417)
        at org.elasticsearch.packaging.test.DockerTests.test010Install(DockerTests.java:103)

```

**Comment by mark-vieira** [comment]
@pugnascotia do you mind taking a look here?

**Comment by mark-vieira** [comment]
Since this is affecting pull requests I've gone ahead and muted this in 8e9f8ba4202432f5c3e84ccdd61ee8499a58603e.

**Comment by pugnascotia** [comment]
Fixed in #60859.
